src/libraries/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -247,6 +244,18 @@ public static IEnumerable<object[]> ProtocolMismatchData()
             yield return new object[] { SslProtocols.Tls12, SslProtocols.Tls11, typeof(AuthenticationException) };
         }
 
+        public static IEnumerable<Object[]> SupportedProtocolData()
+        {
+            if (PlatformDetection.SupportsTls11)
+            {
+                yield return new object[] { SslProtocols.Tls11 };
+            }
+
+            if (!PlatformDetection.SupportsTls12)
```

Comment:
Is the `!` (not) correct here?

Reply:
fixed. I did some experiments and forget it to take it out again ;(